doc/2/controllers/devices/linkAsset/index.md

```diff
@@ -33,7 +33,8 @@ Method: PUT
     "measureNames": [
       {
         "asset": "<name of the measure in the asset>",
-        "device": "<name of the measure in the device>"
+        "device": "<name of the measure in the device>",
+        "type": "<type of the measure in the device>"
```

Comment:
I think we don't need the type in the API, since we have the name then we can retrieve the type

Reply:
as I understand, I remove this from the doc and I need to update measureNames during linkingAssets by adding "type" from deviceModel.device.measures (and not request it in body request).

Reply:
Yes exactly :+1: The code is a little more complex but the API is easier for end users